src/lang/english.txt

```diff
@@ -3992,10 +3992,10 @@ STR_ORDER_SERVICE_AT                                            :Service at
 STR_ORDER_SERVICE_NON_STOP_AT                                   :Service non-stop at
 
 STR_ORDER_NEAREST_DEPOT                                         :the nearest
-STR_ORDER_NEAREST_HANGAR                                        :the nearest Hangar
-STR_ORDER_TRAIN_DEPOT                                           :Train Depot
-STR_ORDER_ROAD_VEHICLE_DEPOT                                    :Road Vehicle Depot
-STR_ORDER_SHIP_DEPOT                                            :Ship Depot
+STR_ORDER_NEAREST_HANGAR                                        :the nearest hangar
+STR_ORDER_TRAIN_DEPOT                                           :Train depot
```

Comment:
I think these end up in the middle of a sentence, in which case they should just be "train depot" etc.

Haven't checked yet.

Reply:
I have considered whether or not to keep these capitalized too! Mainly because the Industry names have to remain capitalized due to this same reason.